source/modules/MEI.header.xml

```diff
@@ -43,6 +43,9 @@
       <rng:optional>
         <rng:ref name="pubStmt"/>
       </rng:optional>
+      <rng:optional>
+        <rng:ref name="availability"/>
```

Comment:
You added `availability` to the `manifestation` element via `macro.bibldescPart`, right? However, in `item` availabilty is added directly (together with physDesc and physLoc). Does it make sense to use the same mechanism with `macro.bibldescPart` there too?

Reply:
@musicEnfanthen I think this would be a different thing. If you think that should be changed, maybe open another issue?

Reply:
I thought about this. If we would add this element directly, we would force an other order of the elements. The model is only used by manifestation so I don't see any problems with this solution, but maybe I'm wrong.
My intention was to keep the order of the elements within manifestation and item analog against each other. Item has just some omittions.

Reply:
OK, so maybe that's soemthing to check for in a future review of this part of the specs ;)